### PR TITLE
Instrument GATK functions (resolves #93)

### LINKED ISSUE #93: Log timings to master for GATK pipeline
I need this for some benchmarking runs.

### PR DESCRIPTION
**Pull request by fnothaft**
Adds timing functions to wrap around GATK processing step calls for benchmarking purposes. Additionally, I've added miscellaneous bugfixes for some of the preprocessing tools.

### CODE REVIEW

File: src/toil_lib/tools/preprocessing.py [hunk 1]
```diff
@@ -84,6 +85,17 @@ def run_samtools_index(job, bam):
     return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'sample.bam.bai'))
 
 
+def _log_runtime(job, start, end, cmd):
+
+    elapsed_time = end - start
+    
+    seconds = int(elapsed_time) % 60
```
Comment (fnothaft) [inline src/toil_lib/tools/preprocessing.py]: This math needs to be rejiggered so that seconds don't get double counted.

**Comment by fnothaft** [comment]
This is good to go from my side. Ping @jvivian for review.

**Comment by jvivian** [comment]
@fnothaft — Sorry for the delay, flying back to the states this weekend. I'll be back at work by Monday.

**Comment by fnothaft** [comment]
Np! Thanks @jvivian! Have a good trip.

**Pull request merged**